sycl/test/basic_tests/reduction_ctor.cpp

```diff
@@ -1,166 +0,0 @@
-// RUN: %clangxx -fsycl %s -o %t.out
-// RUN: %RUN_ON_HOST %t.out
-
-// This performs basic checks such as reduction creation, getIdentity() method,
-// and the combine() method of the aux class 'reducer'.
-
-#include "reduction_utils.hpp"
```

Comment:
This seems to be the last file using `reduction_utils.hpp`. As such it should also be removed.